scripts/serve/serve-static.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable no-console */
+const express = require('express');
+const path = require('path');
+const webpack = require('webpack');
+const MemoryFS = require('memory-fs');
+const mime = require('mime-types');
+
+const compile = (webpackConfig, vfs) => (
+  new Promise((resolve, reject) => {
+    const compiler = webpack(webpackConfig);
+    // setup a virtual file system to write webpack files to.
+    if (vfs) {
+      compiler.outputFileSystem = new MemoryFS();
+    }
+    console.log('[Terra-Toolkit:serve-static] Webpack compilation started');
+    compiler.run((err, stats) => {
+      if (err || stats.hasErrors()) {
+        console.log('[Terra-Toolkit:serve-static] Webpack compiled unsuccessfully');
+        reject(err || new Error(stats.toJson().errors));
+      } else {
+        console.log('[Terra-Toolkit:serve-static] Webpack compiled successfully');
+        resolve([webpackConfig.output.path, compiler.outputFileSystem]);
+      }
+    });
+  })
+);
+
+const generateSite = (site, config, vfs, production) => {
```

Comment:
Although a simple function, it might be worth adding a comment saying this function loads static assets if provided, otherwise will compile for you

Reply:
This whole file needs more comments